### PR TITLE
Initial Relics Implementation

### PR DESCRIPTION
**Pull request by Linkaaaaa**
- Added several relic buffs and icons
- Added skill ids for buffs and hits
- Added icons overrides for the relic hits
- Added damage modifiers for weaver and deadeye relics

### CODE REVIEW

File: GW2EIEvtcParser/EIData/Buffs/CommonBuffs.cs [hunk 1]
```diff
@@ -163,12 +163,29 @@ internal static class CommonBuffs
             new Buff("Gourd Vibrations", GourdVibrations, Source.Gear, BuffClassification.Gear, BuffImages.OrbOfAscension),
             new Buff("Cat's Shadow", CatsShadow, Source.Gear, BuffClassification.Gear, BuffImages.OrbOfAscension),
             // Relics
-            new Buff("Relic of the Dragonhunter", RelicOfTheDragonhunter, Source.Gear, BuffStackType.StackingSomething, 999, BuffClassification.Gear, BuffImages.RelicOfTheDragonhunter), // Applied on target
+            new Buff("Relic Player Buff (Dragonhunter / Isgarren / Peitha)", RelicTargetPlayerBuff, Source.Gear, BuffClassification.Gear, BuffImages.Unknown),
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EIData/Buffs/CommonBuffs.cs:166]: Should be BuffClassification.Hidden, does not serve any visible purpose to the end user.

File: GW2EIEvtcParser/EIData/Buffs/CommonBuffs.cs [hunk 2]
```diff
@@ -163,12 +163,29 @@ internal static class CommonBuffs
...
+            new Buff("Relic of the Dragonhunter", RelicOfTheDragonhunterNPCBuff, Source.Gear, BuffStackType.StackingSomething, 999, BuffClassification.Gear, BuffImages.RelicOfTheDragonhunter), // Applied on target
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EIData/Buffs/CommonBuffs.cs:167]: NPCBuff > TargetBuff, can very well be applied on Players in a WvW context

File: GW2EIEvtcParser/EIData/Buffs/CommonBuffs.cs [hunk 3]
```diff
@@ -163,12 +163,29 @@ internal static class CommonBuffs
...
+            new Buff("Relic of the Firebrand", RelicOfTheFirebrand, Source.Gear, BuffClassification.Gear, BuffImages.RelicOfTheFirebrand),
+            new Buff("Relic of the Herald", RelicOfTheHerald, Source.Gear, BuffStackType.Stacking, 10, BuffClassification.Gear, BuffImages.RelicOfTheHerald),
+            new Buff("Relic of the Scourge", RelicOfTheScourge, Source.Gear, BuffStackType.Stacking, 10, BuffClassification.Gear, BuffImages.RelicOfTheScourge),
+            new Buff("Relic of the Weaver", RelicOfTheWeaver, Source.Gear, BuffClassification.Gear, BuffImages.RelicOfTheWeaver),
+            new Buff("Relic of the Zephyrite", RelicOfTheZephyrite, Source.Gear, BuffClassification.Gear, BuffImages.RelicOfTheZephyrite),
+            new Buff("Relic of Cerus", RelicOfCerusBuff, Source.Gear, BuffClassification.Gear, BuffImages.RelicOfCerus),
+            new Buff("Relic of Dagda", RelicOfDagdaBuff, Source.Gear, BuffClassification.Gear, BuffImages.RelicOfDagda),
+            new Buff("Relic of Isgarren", RelicOfIsgarrenNPCBuff, Source.Gear, BuffClassification.Gear, BuffImages.RelicOfIsgarren),
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EIData/Buffs/CommonBuffs.cs:183]: same comment as dragon hunter relic

Comment (EliphasNUIT) [reply GW2EIEvtcParser/EIData/Buffs/CommonBuffs.cs:183]: I also suspect it should be "BuffStackType.StackingSomething, 999" if it is working like the dragon hunter relic.

**Comment by EliphasNUIT** [comment]
Do you have a log containing the relic of Isgarren? I'd like to verify its type.

File: GW2EIEvtcParser/EIData/Buffs/CommonBuffs.cs (hunk 3 above)
Comment (Linkaaaaa) [reply GW2EIEvtcParser/EIData/Buffs/CommonBuffs.cs:183]: i guess stacking type 2 is "stacking something" ?

**Comment by Linkaaaaa** [comment]
log with 70806 isgarren

[20230826-141433.zip](https://github.com/baaron4/GW2-Elite-Insights-Parser/files/12445320/20230826-141433.zip)

File: GW2EIEvtcParser/EIData/Buffs/CommonBuffs.cs (hunk 3 above)
Comment (EliphasNUIT) [reply GW2EIEvtcParser/EIData/Buffs/CommonBuffs.cs]: Correct, I need to find a better name for the type, so far I don't know how it is different than StackingType 5

**Comment by EliphasNUIT** [comment]
Thanks!

**Pull request merged**